java/src/com/swiftnav/sbp/client/SBPHandler.java

```diff
@@ -201,9 +197,7 @@ protected SBPMessage getNext() {
                     if (msg != null) {
                         return msg;
                     }
-                    continue;
-                } catch (InterruptedException e) {
-                    continue;
+                } catch (InterruptedException ignored) {
```

Comment:
I think SonarCloud flagged this because you touched this line, if it makes sense we can just fix according to SonarCloud recommendation, if it doesn't make sense we can ignore the warning.

![image](https://user-images.githubusercontent.com/183436/161934882-b61d733c-42b0-4b1e-a481-34ece2f7a20b.jpeg)

Reply:
hmm yea, maybe we could log the error as it suggests but is also another thing which the library doesn't look like it uses a logger instead using System.out.println() everywhere

I don't think it should have to throw an InterruptedException and it follows logically like previously:
https://github.com/swift-nav/libsbp/runs/2882927558

Reply:
I think we should ignore it (with `// NOSONAR` I think) and make a ticket to track switching to a logging facade